examples/with-typescript/pages/index.tsx

```diff
@@ -1,8 +1,8 @@
-import * as React from 'react'
```

Comment:
in my experience every TS file that uses JSX must have this exact line, because the JSX definitions are defined in the react types.